streams/src/test/java/org/apache/kafka/streams/state/internals/RocksDBWindowStoreTest.java

```diff
@@ -1368,24 +1368,6 @@ public void shouldThrowNullPointerExceptionOnRangeNullToKey() {
         windowStore.fetch(1, null, ofEpochMilli(1L), ofEpochMilli(2L));
     }
 
-    @Test
-    public void shouldNoNullPointerWhenSerdeDoesNotHandleNull() {
```

Comment:
Removed this test entirely as it no longer makes sense when the RocksDB store can't and shouldn't interact with the serdes at all